pandas/io/formats/format.py

```diff
@@ -1955,17 +1955,15 @@ def __init__(self, accuracy: int | None = None, use_eng_prefix: bool = False):
     def __call__(self, num: int | float) -> str:
         """
         Formats a number in engineering notation, appending a letter
-        representing the power of 1000 of the original number. Some examples:
-
-        >>> format_eng(0)       # for self.accuracy = 0
+        representing the power of 10 of the original number. Some examples:
+        >>> format_eng = EngFormatter(accuracy=0, use_eng_prefix = True)
+        >>> format_eng(0)
         ' 0'
-
-        >>> format_eng(1000000) # for self.accuracy = 1,
-                                #     self.use_eng_prefix = True
+        >>> format_eng = EngFormatter(accuracy=1, use_eng_prefix = True )
+        >>> format_eng(1000000)
```

Comment:
In all Python versions we support we can use `1_000_000` for one milion, which is easier to read (faster to count the nunber of zeros). Probably worth using it.

Reply:
Didn't know about this one thanks!